fc/cli/main.py

```diff
@@ -29,3 +29,35 @@ def task(title, description, parent_story, username, in_progress, no_assign):
             click.echo('Triage task put into In Progress')
     except HTTPError as exception:
         click.echo('{} task creation failed with {}'.format(new_task.type_str(), exception))
+
+@cli.command()
+def triagesearch():
+    click.echo('searching for triage tasks')
+
+    auth = ComboAuth(None)
```

Comment:
We will need to continue to support the `--username` paradigm for all the commands and pass it into `ComboAuth`.  This also applies to the command added below.